src/condor_tests/test_cgroup_oom.py

```diff
@@ -163,6 +164,37 @@ def slice_scope_test_job(condor, slice_scope_job_hash):
     )
     return ssj
 
+@action
+def memory_detection_job_hash(test_dir, path_to_python):
+    # A job which runs condor_config_val DETECTED_MEMORY.
+    # This should return the memory limit of the cgroup
+    bin_dir = htcondor2.param["BIN"]
```

Comment:
Note: this gets the BIN directory of the bindings being used to run the test, which could at least theoretically be different than the BIN directory of the condor under test.  I note this mostly because if you added `condor` as a fixture-argument to this function, you could do

```
bin_dir = condor.run_command( ['condor_config_val', 'BIN'] ...)
```

or so, and not have to use htcondor2 explicitly at all.  (There should probably be a better way to get config knobs back out of the-condor-under-test.)